scripts/compute_affected_tests.sh

```diff
@@ -15,7 +15,7 @@
 BAZEL_BINARY=$1
 
 # Reference: https://stackoverflow.com/a/6245587.
-current_branch=$(git rev-parse --abbrev-ref HEAD)
+current_branch=$(git branch --show-current)
```

Comment:
Why was this changed? 
https://github.com/oppia/oppia-android/blob/86aa93f69ac54bdda5ca37c9f7d38c2a9b65e399/scripts/compute_affected_tests.sh#L18

I think this should be reverted to match the develop branch.

Reply:
@rt4914 I have reverted it.